app/api/hypercerts/[hypercertId]/image/route.ts

```diff
@@ -78,7 +78,7 @@ export async function GET(
     const imageOrUrl = res.hypercerts.data?.[0]?.metadata?.image;
 
     // Use placeholder image if no image URL or data is found
-    if (!imageOrUrl) {
+    if (!imageOrUrl || imageOrUrl === "https://hypercerts.org/logo.png") {
```

Comment:
Should this be the SVG?

Reply:
no, its seems the api returns this logo when an image is not found for the hypercert, however we want to show the svg from our static files instead.